src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -493,10 +493,13 @@ def aks_create(
     enable_addon_autoscaling=False,
     enable_cilium_dataplane=False,
     custom_ca_trust_certificates=None,
+    # advanced networking
     enable_advanced_network_observability=None,
     advanced_networking_observability_tls_management=None,
     enable_fqdn_policy=None,
     enable_acns=None,
+    disable_acns_observability=None,
+    disable_acns_security=None,
```

Comment:
If this is not enabled by default, maybe you just need to add the options to disable this feature in the `aks update` command

Reply:
Updated

Reply:
they are still there :)

Reply:
customers can create an acns cluster with `--enable-acns` with `--disable-acns-observability` or  `--enable-acns` with `--disable-acns-security` so i left them under the `aks create` command

Reply:
Oh, I misunderstood this. Now I see what you mean.